styles/navbar.scss

```diff
@@ -99,14 +99,19 @@
       transform: translate(-50%, -50%);
       display: flex;
 
+      [dir='ltr'] & {
+        left: 33px;
+        right: unset;
+      }
```

Comment:
Changing the order will cause the nav icon to always appear on the left, shouldn't be set up that way.